index.bs

```diff
@@ -549,18 +510,30 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
 
       <p>
         The <dfn for='AudioConfiguration' dict-member>contentType</dfn> member
-        represents the MIME type of the audio track.
+        represents the [=MIME type=] of the audio track.
       </p>
 
       <p>
-        To check if a {{AudioConfiguration}} <var>configuration</var> is a
+        To check if a {{AudioConfiguration}} |configuration| is a
         <dfn>valid audio configuration</dfn>, the following steps MUST be run:
         <ol>
           <li>
-            If <var>configuration</var>'s {{AudioConfiguration/contentType}} is
-            not a <a>valid audio MIME type</a>, return <code>false</code> and
+            Let |mimeType| be the result of running [=parse a MIME type=]
+            with |configuration|'s {{AudioConfiguration/contentType}}.
+          </li>
+          <li>
+            If |mimeType| is <code>failure</code>, return <code>false</code> and
```

Comment:
```suggestion
            If |mimeType| is failure, return `false` and
```

Reply:
Markup shortcuts don't work in algorithm blocks (at least for me).